plugin/mlsdbonlinelocator.cpp

```diff
@@ -79,7 +79,7 @@ void MlsdbOnlineLocator::defaultVoiceModemChanged(const QString &modem)
 
 bool MlsdbOnlineLocator::haveFieldData(const QList<MlsdbProvider::CellPositioningData> &cells)
```

Comment:
Would be nice to document in the .h file or so what is the expected behaviour of this, when it should return true and when false.